src/package_source/in_memory.rs

```diff
@@ -108,6 +108,10 @@ impl PackageSource for InMemoryRegistrySource {
 
         Ok(entry.contents.clone())
     }
+
+    fn fallback_sources(&self) -> anyhow::Result<Vec<PackageSourceId>> {
+        todo!("Implement in-memory fallback sources");
```

Comment:
Can we use `anyhow::bail` here? There's no reason to panic since this is already a fallible function

Reply:
Is there something wrong with a panic here? I'm not sure I understand why a bail would be preferable to a panic since this is code that shouldn't be running anyway.